examples/vic_basic/vic_basic.ino

```diff
@@ -1,8 +1,7 @@
 #include "vic.h"
 
-/***************************************
-MAKE SURE YOU HAVE SET '\n' LINE ENDINGS
-***************************************/
+/* You can use Arduino serial monitor (in that case set line endings with line
+   feed) or vic-picocom (find it in shell/) for talking to vic */
```

Comment:
@vargac it may be better to say `which can be found in shell/` rather than `find it in shell/`